### ISSUE TITLE
Edit Event Start and End times are incorrect

### ISSUE THREAD

**Issue by stevejgordon** [HTBox/allReady#1223]
When clicking to edit an event the edit form date time is displaying the incorrect time. It appears the time is being converted to UTC rather than retaining the original local time.

This might be easier to fix after merge of #1154

**Time in event display page (correct):**
![image](https://cloud.githubusercontent.com/assets/3669103/18232450/dc074d48-72c7-11e6-9ece-92ed97f399f8.png)

**Time in event edit page (incorrect):**
![image](https://cloud.githubusercontent.com/assets/3669103/18232452/e32e0fe4-72c7-11e6-8738-ee7dcd52c0e3.png)

**Comment by mgmccarthy** [comment]
@stevejgordon just rebased for #1154 and waiting on merge. Not too sure if it will fix the problem, as any work with DateTimeOffset was abstracted away with #1154, so the problem may still persist. If it does persist, I'd be happy to pick this up.

**Comment by VishalMadhvani** [comment]
@mgmccarthy iirc this is a locale issue. I think the date under the task is being rendered by razor and is in american format mm/dd/yyyy, where as the dates in the form are rendered in JS using UK format dd/mm/yyyy.

you may not be able to reproduce it due to your systems locale settings.

**Comment by tonysurma** [comment]
@dpaquette is this related to any work you are doing?

**Comment by mgmccarthy** [comment]
@tonysurma, my best guess is this is related to a bug that @dpaquette fixed last week sometime that fixed how the TimeTagHelper worked. Based on Dave's changes, this bug should be fixed.

**Comment by stevejgordon** [comment]
@tonysurma I can pull the latest and test it quickly.

**Comment by stevejgordon** [comment]
Looks like start date/time still has issue for me...
![image](https://cloud.githubusercontent.com/assets/3669103/19966740/0f38f640-a1c5-11e6-8309-77d790adcc44.png)
![image](https://cloud.githubusercontent.com/assets/3669103/19966747/16fabbd4-a1c5-11e6-8e2d-8964f8c5b965.png)

**Comment by dpaquette** [comment]
I thought I tested this in my local branch recently and it was working as expected. I will review. The problem is with aligning the formatting used by the bootstrap datetime component and the formatting used by the server.

**Comment by dpaquette** [comment]
I will have this fixed as part of PR #1416

**Comment by dpaquette** [comment]
Can this be assigned to me? This is related to the javascript datetime formatting issue I mentioned in the standup today.

Note this was not fixed as part of PR #1416. That PR grew out of control so decided to split it out.

**Comment by dpaquette** [comment]
Confirmed this with @stevejgordon today. This is partially fixed in that the times are no longer being shifted but the month and date are still being swapped around. I need to do some more testing to see if that is a problem or not.
